app/channels/room_channel.rb

```diff
@@ -95,7 +92,14 @@ def render_error_json(message)
     end
 
     def exit_room
-      UserRoomLog.where(user: current_user, room: @room, exit_at: nil).update(exit_at: Time.now.utc)
+      ActiveRecord::Base.transaction do
+        logs = UserRoomLog.where(user: current_user, room: @room, exit_at: nil)
+        if logs.present?
+          logs.update(exit_at: Time.now.utc)
```

Comment:
__[RuboCop]__

Rails/SaveBang: Use `update!` instead of `update` if the return value is not checked.


<!-- Comment by [SideCI](https://sideci.com) -->